build.gradle

```diff
@@ -311,14 +311,34 @@ configure(project(':desktop')) {
 
 
 configure(project(':monitor')) {
-    mainClassName = 'bisq.monitor.MonitorMain'
+    mainClassName = 'bisq.monitor.Monitor'
+
+    test {
+        useJUnitPlatform()
+        testLogging {
+            events "passed", "skipped", "failed"
+        }
+    }
 
     dependencies {
-        compile project(':core')
-        compile "com.sparkjava:spark-core:$sparkVersion"
-        compile 'net.gpedro.integrations.slack:slack-webhook:1.1.1'
+        compile 'org.slf4j:slf4j-api:1.7.22'
+        compile 'ch.qos.logback:logback-core:1.1.10'
+        compile 'ch.qos.logback:logback-classic:1.1.10'
```

Comment:
Since these dependencies are now repeated between `:monitor` and `:common`, their version numbers can be extracted to `$sf4jVersion` and `$logbackVersion` to avoid version drift between the two subprojects.

Reply:
agreed. However, as the next steps for the Monitoring Node is integrating Bisq Protocol foo, I kept the dependencies separate as the :monitor project will depend on :common eventually.

However, done.